docs/output.md

```diff
@@ -216,6 +222,23 @@ Output Summaries:
 
 [Bakta](https://github.com/oschwengers/bakta) is a tool for the rapid & standardised annotation of bacterial genomes and plasmids from both isolates and MAGs. It provides dbxref-rich, sORF-including and taxon-independent annotations in machine-readable JSON & bioinformatics standard file formats for automated downstream analysis. The output is used by some of the functional screening tools.
 
+### Functional classifications
+
+[InterProScan](#interproscan)
+
+#### InterProScan
+
+<details markdown="1">
+<summary>Output files</summary>
+
+- `interproscan/`
+  - `<samplename>_cleaned.faa`: clean version of the fasta files (amino acids) generated by one of the annotated tools. These contain sequences with no special character
```

Comment:
```suggestion
  - `<samplename>_cleaned.faa`: clean version of the fasta files (amino acids) generated by one of the annotated tools. These contain sequences with no special character.
```

What is an 'annotated tool'? Is this  atool within interproscan?
What is a special character?

Reply:
Ive explained it a bit more now ..